server/migrations/30_add_traces.up.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+CREATE TABLE "otlp_traces" (
+  "trace_id" varchar not null primary key,
+  "tenant_id" uuid,
+  "trace" JSONB,
+  "created_at" timestamp NOT NULL DEFAULT NOW()
```

Comment:
this field gets autofilled on create. the rows are always inserted. for updates we use a delete+insert approach so we'll always have the correct value.

I think this might be useful for deleting old records from time to time